src/wpt/harness.ts

```diff
@@ -439,21 +705,14 @@ function prepare(env: Env, options: TestRunnerOptions): void {
   globalThis.errors = [];
   globalThis.testOptions = options;
   globalThis.env = env;
-  globalThis.promises = {};
+  globalThis.promises = [];
 }
 
 async function validate(
   testFileName: string,
   options: TestRunnerOptions
 ): Promise<void> {
-  for (const [name, promise] of Object.entries(globalThis.promises)) {
-    try {
-      await promise;
-    } catch (err) {
-      globalThis.errors.push(new AggregateError([err], name));
-    }
-  }
-
+  await Promise.all(globalThis.promises);
```

Comment:
If this will never throw (since you're catching them while creating), can you add a comment here?